### PR TITLE
Feature/tests

### LINKED ISSUE #5: 🧪 Write tests for User class

### LINKED ISSUE #6: 🧪 Write tests for Traveler class

### LINKED ISSUE #7: Write tests for Agent class

### LINKED ISSUE #8: Write test data file

### PR DESCRIPTION
**Pull request by nichelicorn**
# Pull Request Template
## Description
* Write test file for User class
* Update test data to match expected API data
🚧 Need to add sad path testing

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by nichelicorn** [comment]
✨ Initial push of User test file
🚧 Still need to add sad path testing for potential error states

File: src/User.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class User {
+  constructor() {
+
+  }
```
Comment (rachael-t) [inline src/User.js:4]: What properties were you thinking of adding?

File: test/user-test.js [hunk 2]
```diff
@@ -1 +1,136 @@
...
+      let tripCost = user13.viewTripCost("2020/10/17");
+
+      expect(tripCost).to.equal(2820);
+    })
+
+    describe("A method to calculate the agent fee for all trips", () => {
+      // should be able to calculate 10% agent fee for all of the above
+      it("should return the agent fee for any trip", () => {
+        let agentFee1 = viewAgentFee(13, "2020/10/17");
+        let agentFee2 = viewAgentFee(7, "2020/10/14");
```
Comment (rachael-t) [inline test/user-test.js:127]: Will `viewAgentFee()` be a method on a class?

**Review by rachael-t — COMMENTED** [review]
Great job building this out so far and having a TDD approach! The first describe block for testing is for "The User properties" and is unique to the User class. I haven't looked at the classes, but something to think about is that the methods you testing/creating will apply to _all_ instances of users, regardless if they are a traveler or an agent. 

When you are testing in child classes (Traveler/Agent) you don't need to test the inherited properties as thoroughly, but you can just test that the class is in fact receiving the inherited data. Play around with this idea and let me know if you have any questions!

File: src/User.js (hunk 1 above)
Comment (nichelicorn) [reply src/User.js:4]: * The User class will have an id, name, username and password
* These will be inherited by the Traveler and Agent classes

File: test/user-test.js (hunk 2 above)
Comment (nichelicorn) [reply test/user-test.js:127]: * Yes. I want this method to work for travelers and agents, so I included it in the User class

**Comment by nichelicorn** [comment]
* Push up remaining test files

**Pull request merged**